code/modules/jobs/job_types/captain.dm

```diff
@@ -15,6 +15,7 @@
 	exp_requirements = 1200
 	exp_type = EXP_TYPE_COMMAND
 	exp_type_department = EXP_TYPE_COMMAND
+	special_notice = "You are not a security officer. Do not do their job."
```

Comment:
I'd replace this with something more inline with our policies on Captains: "You are not a frontline security officer. Let the frontline officers handle the arrests unless they need your help or they are massively understaffed." as we do allow captains to perform arrests if needed, but not ignore their job to just be sec officer++.

That said, the original message is perfect for the HoP.

Reply:
Updated it.